tensorflow_addons/losses/python/focal_loss.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Focal loss"""
```

Comment:
So I see still there being style issues. If I checkout your repo and run `docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-custom-op make code-format` a bunch of style changes are performed. Could you please try that

Reply:
I did this again and here is my output:
```
--- Format check step 1 of 3:  do_bazel_config_format_check (Check Bazel file format) ---

do_bazel_config_format_check will NOT run due to the absence of code changes.


--- Format check step 2 of 3:  do_python_format_check (Check python file format) ---

do_python_format_check will NOT run due to the absence of code changes.


--- Format check step 3 of 3:  do_clang_format_check (Check  C++ file format) ---

do_clang_format_check will NOT run due to the absence of code changes.

---- Summary of format check results ----
1. do_bazel_config_format_check: Check Bazel file format
  PASS
2. do_python_format_check: Check python file format
  PASS
3. do_clang_format_check: Check  C++ file format
  PASS

0 failed; 3 passed.

```